packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -45,6 +48,9 @@ export class GuxFormFieldColor {
   @Prop()
   labelPosition: GuxFormFieldLabelPosition;
 
+  @Prop()
+  indicatorMark: GuxFormFieldIndicatorMark = 'required-only';
```

Comment:
If `required` on the input is true will that means `indicatorMark` is also always true after this change is made? I'm just trying to make sure I understand all of the logic here and what the purpose of both `required` on the input and `indicatorMark` being used instead of only just `required` on the input?

Reply:
I discussed this with Daragh previously and we agreed that while it kind of looks obvious to make this a boolean property, doing so would make it much more difficult if an additional style needed to be implemented further down the line. 

Since 'required-only' is the default, this will not impact anyone currently using this component but will also support the scenario where the (optional) mark is shown. 

The required on the input will have a bearing on the display depending on what's chosen for the indicator mark. 

Here's a rough list of scenarios: 
input === 'required' && indicator-mark === 'optional-only' then do not show asterisk
input !== 'required' && indicator-mark === 'optional-only' then show (optional)

input === 'required' && indicator-mark === 'required-only' then show asterisk
input !== 'required' && indicator-mark === 'required-only' then  do not show (optional)

Reply:
Makes sense, thanks!

Reply:
Is there any way to document those scenarios somewhere in the component so it's more obvious?

Reply:
@jason-evans-genesys I added a comment to the prop in the field and I think it's maybe a little clearer in the updated implementation now. Let me know if you think I could add in some more comments and stuff here